sections/algorithmic_components.tex

```diff
@@ -1155,64 +1168,27 @@ \subsection{Association and Matching}
+Cross-matching may be undertaken using the Optimistic Pattern Matching (B) algorithm of Tabur \cite{tabur2007}. The algorithm defines an order $m$ (default 6) size $m-1$ acyclic connected tree as the pattern to match between catalogs.  Trees are constructed and matched for the brightest $n$ stars (default = 50) in descending order until a match is found.
 
-\begin{itemize}
-\item An optical distortion model is subtracted from the positions of
-  the detected sources on the sensor to define a gnomonic plane projection.
-\item Detected sources are ordered in descending brightness and the
-  brightest $n$ stars (default = 50) are selected, $I$.
-\item Given the extent of the sensor (defined from the initial WCS)
-  the brightest $n$ sources are extracted from the reference catalog, $R$.
-\item The pairwise distances and position angles between the reference
-  catalog sources, $R$, are measured (generating $n(n-1)/2$ pairs).
-\item For the $m$ brightest sources in $I$ the length of the edges and
-  the position angles of the edges of the graph are calculated (with
-  the brightest source the center of the graph). Angles are relative
-  to the sensor orientation and distances are in angular coordinates.
-\item A binary search identifies all pairs in $R$ with a length
-  matching the length of the edge containing the brightest two sources
-  (within a tolerance that has a default of 3 arcsec).  The difference
-  in position angle between the reference and source edges is assumed
-  to be due to the rotation of the sensor relative to the reference
-  catalog and this position angle difference is accounted for in all
-  remaining matches.
-\item The remaining $m-2$ edges are compared to the pairs in $R$ and a
-  match is assumed if the edge length and postion angles are within
-  the tolerances (defaults 3 arcsec and 1 degree respectively).
-\item If $m-1$ matches are not found the search repeats; initially for
-  the remaining matches to the length of the edge corresponding to the
-  brightest two sources and then for a new graph that excludes the
-  brightest object from $I$ and finds the next brightest $m$ sources.
-\end{itemize} 
-
-Once $m-1$ edges match, an initial verification step is performed where a gnomonic projection of the reference catalog is fit to the matched stars. Given a consistent fit to the gnomonic projection all sources and reference objects are matched in sensor coordinates (sorted by brightness) with the brightest source in $I$ selected when two sources are within the match tolerance (default 3 arcsec). 
+After a matched solution is found, we will verify the quality of the match by comparing the positions of the science and reference objects.
 
 For the case of no WCS for the sensor or a significant error in the
-WCS ($> 3$ arcsec), a blind matching will be undertaken using the
-algorithms in astrometry.net. 
+WCS ($> 3$ arcsec), blind matching will be undertaken (e.g., by using the
```

Comment:
I wonder if we should just take this out.  Right now Chris's algorithm is matching for offsets up to 30 arcsec and I don't know if we want a dependence on a.net in the design document.  If you think we need to hedge our bets on blind solving, we could just say "a blind solver" rather than specifying a.net specifically.